en/docs/design/api-policies/samples/create-custom-cc-policy.md

```diff
@@ -0,0 +1,215 @@
+# Create a Custom Choreo Connect Policies
+
+Choreo Connect supports the following operations and the policies supporting these operations are shipped with the WSO2 API Manager by default. The following table contains the action name and parameters of the operations that are supported in Choreo Connect.
+
+
+<table>
+ <tr>
+    <th>Action Name</th>
+    <th>Parameters</th>
+    <th>Applicable Flows</th>
+    <th>Description</th>
+ </tr>
+ <tr>
+    <td>SET_HEADER</td>
+    <td>
+        <ul>
+            <li>headerName</li>
+            <li>headerValue</li>
+        </ul></td>
+    <td>Request</td>
+    <td>Set a header in the request flow.</td>
+ </tr>
+ <tr>
+    <td>REMOVE_HEADER</td>
+    <td>headerName</td>
```

Comment:
Shall we add a <ul> <li> here also even though it is one item since we have added it in other places

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/5528/commits/4993fd03bc900bab19a04334c02c9d78e9e1df14